### DISCUSSION TITLE
FTC Model design

### DISCUSSION THREAD

**Discussion by monyhm** [FIRST-Tech-Challenge/FtcRobotController#752] [General] [answered]
Where can I find the robot design?

**Comment by cmacfarl** [comment, marked as answer]
Start here: https://ftc-docs.firstinspires.org/en/latest/

Take a look at the New Team Resources page where you will find a link to
the REV Kickoff Concepts documentation which may contain what you are
looking for.

But in general there's a lot of good stuff on ftc-docs.
